scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/policy/ram/RamCompute.java

```diff
@@ -0,0 +1,39 @@
+package org.ow2.proactive.scheduler.policy.ram;
+
+import java.io.Serializable;
+import java.lang.management.ManagementFactory;
+
+import org.objectweb.proactive.core.runtime.LocalNode;
+import org.objectweb.proactive.core.runtime.ProActiveRuntimeImpl;
+
+import com.sun.management.OperatingSystemMXBean;
+
+
+public class RamCompute implements Serializable {
+
+    private static final long GIGABYTE = (1024L * 1024L * 1024L);
+
+    public RamCompute() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1186)